source/HiringPanel.cpp

```diff
@@ -46,7 +47,8 @@ void HiringPanel::Step()
 void HiringPanel::Draw()
 {
 	const Ship *flagship = player.Flagship();
-	const Interface *hiring = GameData::Interfaces().Get("hiring");
+	// Tweak planet panels for small screens by 60px to the left.
```

Comment:
I think the code is pretty self-explanatory:
"If the width of the screen is less than 1280, get the interface named 'hiring (small screen)', otherwise, get the interface named 'hiring'."
This comment is about the content of those interface definitions, which I'd rather not describe here.
```suggestion
```